apps/site/pages/docs/building-sections/overrides/migration-v1-to-v2.mdx

```diff
@@ -1 +1,41 @@
-how to migrate from v1 to v2
\ No newline at end of file
+import { Steps, Callout } from 'nextra-theme-docs'
+
+<header>
+# Migrating Overrides v1 to v2
+</header>
+
+Overrides v2 offers a more flexible and customizable approach to component overrides. It's strongly recommended to update your component customization methods accordingly.
+
+<Callout type="warning" emoji="⚠️">
+If you choose to continue using Overrides v1, please refer to the [Overriding components (v1)](/docs/building-sections/overrides/overrides-version-1) and [Creating a new section](/docs/building-sections/overrides/overrides-version-1-new-section) guides.
+</Callout>
+
+## Main differences
+
+| Feature                        | Overrides v1                           | Overrides v2                           |
+| ------------------------------ | -------------------------------------- | -------------------------------------- |
+| Overrides per section         | One override per section.               | Multiple overrides per section.         |
+| Native section                | The new component overrides the native one. | The new component is based on the native one and can coexist with it. |
+| Headless CMS property schema  | No customization, limited to native schemas. | Customizable schema allowing definition of new properties. |
+| Feature availability          | Experimental feature and may not function as expected. | Stable feature. |
+
+## Step by step
+
+<Steps>
+
+### Update `@faststore/core` Package
+
+Ensure your store's `@faststore/core` package is version 3.x or higher to utilize Overrides v2. Refer to the [release note](https://github.com/vtex/faststore/releases/tag/v3.0.0) for updating instructions.
+
+### Review Existing Overrides
+
+Identify all overrides in your codebase based on Overrides v1.
+
+### Add the steps on how to migrate to the new version 
+
```

Comment:
@eduardoformiga @hellofanny I still not sure of the entire process to migrate from overrides v1 to v2. Can you help here?
Once the person updates the core, what else should be checked to make sure the migration is complete?:

- Review existing overrides: once identified, what should the person do to have them in the v2? Create a new one but following the v2 tutorial?
- ...

Reply:
Basically, the changes come from this PR:
- https://github.com/vtex/faststore/pull/2091

previously we passed the name of the section as a string to the `section` key in the overrides object from `getOverriddenSection` method, for example:
```js
getOverriddenSection({
  section: 'Alert',
  components: {
     Alert: { Component: CustomAlert },
  },
}),
```

now we import the Section itself and passed to the Section key (S uppercase) in the overrides object from `getOverriddenSection` method, for example:

```js
import { AlertSection, getOverriddenSection } from "@faststore/core";

const NewAlert = getOverriddenSection({ 
  Section: AlertSection, 
  components: {
    Alert: { Component: CustomAlert },
  }, 
});
```